### DISCUSSION TITLE
Destination folder

### DISCUSSION THREAD

**Discussion by qupear** [Alex313031/Thorium-Win#52] [General]
It is installed in appdata, but could I specify another installation folder somehow?

**Comment by Alex313031** [comment]
@qupear To install it at the system level (in Program Files) you can append `--system-level` to the thorium_mini_installer.exe

If you want it in another place, you could download the portable .zip, unpack it someplace, and then make a shortcut to thorium.exe and place the shortcut on the desktop, taskbar, start menu.
